app/assets/stylesheets/dm/components/_profile.scss

```diff
@@ -35,3 +35,20 @@
   position: relative;
   text-decoration: none !important;
 }
+
+.profile-h1 {
```

Comment:
sorry to do this to yah... but for scaling and mobile responsiveness sake, can we use the uswds styles for all of these? for instance place these classes on the h1 in question? https://designsystem.digital.gov/utilities/font-size-and-family/
the units don't have to be exact, round to the closest possible

Reply:
unless it didn't work then please let me know :)

Reply:
updated to use `uswds` styles plus forcing the font family